docker/init.sls

```diff
@@ -61,6 +63,8 @@ docker-config:
       - pkg: docker-package
     - watch_in:
       - service: docker-service
+    - context:
+        config: {{ docker.config }}
```

Comment:
maybe `config: {{ docker.config|json }}` or `config: {{ docker.config|yaml }}` is safer.

Reply:
We need to avoid this issue: https://docs.saltstack.com/en/develop/topics/releases/2019.2.0.html#non-backward-compatible-change-to-yaml-renderer

Reply:
I'll check it with my Py3 system and will add this for py3 uses. Good catch.

Reply:
Although docs recommend `|tojson` the community seems to prefer `|json` for backwards compatibility.  If you expect complicated configuration, including comments, then `|yaml` may be appropriate.  But in this PR I believe `|json` is sufficient.